project/project-info.conf

```diff
@@ -174,4 +174,20 @@ project-info {
       }
     ]
   }
+  akka-projection-dynamodb: ${project-info.shared-info} {
+    title: "Akka Projection DynamoDB"
+    jpms-name: "akka.projection.dynamodb"
+    levels: [
+      {
+        readiness: Supported
+        since: "2024-10-16"
+        since-version: "1.6.0"
+      }
+      {
+        readiness: Incubating
+        since: "2024-05-23"
+        since-version: "0.1"
+      }
```

Comment:
might be confusing with incubating here, since the version numbering doesn't match akka-projection

supported date is ok even though not released at that date, if we remember we could touch it when releasing

Reply:
removed incubating level